src/arvgvfakecamera.c

```diff
@@ -467,9 +482,11 @@ arv_gv_fake_camera_start (ArvGvFakeCamera *gv_fake_camera)
 			g_clear_object (&inet_socket_address);
 			g_clear_object (&socket_address);
 
+                        /* global broadcast discovery */
 			inet_address = g_inet_address_new_from_string ("255.255.255.255");
 			discovery_address_string = g_inet_address_to_string (inet_address);
 			arv_debug_device ("[GvFakeCamera::start] Discovery address = %s", discovery_address_string);
+			g_free (discovery_address_string);
 			inet_socket_address = g_inet_socket_address_new (inet_address, ARV_GVCP_PORT);
 			if (g_strcmp0 (gvcp_address_string, discovery_address_string) == 0)
```

Comment:
Use of discovery_address_string after free

Reply:
Oops!